.github/actions/publish-image/action.yml

```diff
@@ -0,0 +1,57 @@
+name: "Publish Image to domain"
+description: "Publish Image to quay.io or dockerhub or another domain - only publish the version on master - elsewhere version and branch"
+
+inputs:
+  domain:
+    description: 'domain to publish image to'
+    required: true
+    type: string
+  username:
+    description: 'login username'
+    required: true
+    type: string
+  password:
+    description: 'login password'
+    required: true
+    type: string
+  dry-run:
+    description: dry run flag
+    required: true
+    type: boolean
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get docker image tag name
+      shell: bash
+      run: |
+        if [[ $BRANCH_NAME == "master" ]]; then
+            TAG_VERSION=$(grep -m1 version package.json | awk '{ print $2 }' | sed 's/[", ]//g')
+        else
+            TAG_VERSION=$BRANCH_NAME-${{ github.run_id }}
+        fi;
+        echo "TAG_VERSION=$TAG_VERSION" >> $GITHUB_ENV
+    - name: Publish image
+      shell: bash
+      run: |
+        DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+        cd $DIR/../../../
```

Comment:
why you need this here?

Reply:
I guess this should enter the correct folder before installing but I'll check again

Reply:
Okay it is not required, deleted